aspnet/tutorials.md

```diff
@@ -7,7 +7,6 @@ ms.assetid: fddf8e5e-c055-4f81-b691-78deef871112
 ms.author: aspnetcontent
 ms.date: 09/20/2017
 ms.technology: aspnet
-uid: aspnet/tutorials
```

Comment:
is this a dup of a non-core tutorials page?  If so, rather than remove the uid altogether, maybe we could just change it to aspnetcore/tutorials, then we can still use xref links to it.

Reply:
duplicate UID